tests/performance/performance_suite_test.go

```diff
@@ -59,3 +66,81 @@ var _ = AfterSuite(func() {
 	err = globalhelper.RemoveContentsFromReportDir()
 	Expect(err).ToNot(HaveOccurred())
 })
+
+func ConfigurePrivilegedServiceAccount(namespace string) error {
```

Comment:
For consistency with the other test suites it might make sense to move this and getPrivilegedServiceAccountObjects into performance/helper/helper.go, since the x_suite_test files usually just contain the TestX function and Before and After Suite vars